product-service/src/libs/dbHelper.ts

```diff
@@ -0,0 +1,15 @@
+import {Client, ClientConfig} from 'pg';
+
+const { PG_HOST, PG_PORT, PG_DB, PG_USERNAME, PG_PASSWORD } = process.env;
+
+const dbClient: ClientConfig = {
+  host: PG_HOST,
+  port: +PG_PORT,
```

Comment:
what is this +?

Reply:
There was an error with config. It requires type number, but I declared port as string. So, the + means "do me number type from string"

Reply:
BTW, regarding your comment in the RS app. You couldn't see the removal part because I changed my project from the previous task. I decided to change architecture after QA session :)